app/assets/javascripts/javascripts/smooth_anchor_links.js

```diff
@@ -1,10 +1,14 @@
-module("anchor_links", function (module) {
-  module.exports = (function(){
+var smoothAnchorLinks = function(){
+  var init = function(){
     $('a[href^="/#"]').click(function () {
       $("html, body").animate({
         scrollTop: $('[name="' + $.attr(this, 'href').substr(2) + '"]').offset().top - 90
       }, 500);
       return false;
     });
-  })();
-});
+  }
+
+  return {
+    init: init
+  }
+}();
```

Comment:
Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.